docs/getting-started/examples.md

```diff
@@ -169,3 +169,19 @@ with Diagram("Stateful Architecture", show=False):
 ```
 
 ![stateful architecture diagram](/img/stateful_architecture_diagram.png)
+
+## Using Custom Images
```

Comment:
Please use a more practical example like this: https://github.com/mingrammer/diagrams/issues/22#issuecomment-586954430

Reply:
For sure. I added a more meaningful example.